package.json

```diff
@@ -30,7 +30,7 @@
     "uuid": "^3.3.2"
   },
   "scripts": {
-    "start": "chmod +x ./init.sh && ./init.sh && cp env-config.js ./public/ && npm run build:css && react-scripts start",
```

Comment:
How is this related to the ticket?

Reply:
It is not related in any way but the reason why I changed that was due to the fact the application would not running with that included.

Reply:
That requires a separate ticket and pull request.

Reply:
I am going to let the sprint lead know and we create the ticket.

Reply:
If it was addressed in that ticket, then what is this change still doing in your pull request?

Reply:
I have rebased with master and fixed that issue.